esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -401,10 +411,20 @@ void EsdlServiceImpl::configureTargets(IPropertyTree *cfg, const char *service)
             m_methodCRTransformErrors.remove(method);
             m_customRequestTransformMap.remove(method);
 
-            IPropertyTree * customRequestTransform = nullptr;
+            Owned<IPropertyTree> methodCrt;
             try
             {
-                customRequestTransform = methodCfg.queryPropTree("xsdl:CustomRequestTransform");
+                methodCrt.set(methodCfg.queryPropTree("xsdl:CustomRequestTransform"));
+                if (!methodCrt)
```

Comment:
Would it make sense to detect both the single transform and the list of transforms, at least to give feedback that the list will be ignored?